data-processing-lib/ray/src/data_processing_ray/runtime/ray/transform_orchestrator.py

```diff
@@ -135,6 +135,7 @@ def orchestrate(
         # Compute execution statistics
         logger.debug("Computing execution stats")
         stats = runtime.compute_execution_stats(ray.get(statistics.get_execution_stats.remote()))
+        stats["processing_time"] = round(stats["processing_time"], 3)
```

Comment:
we use the string "processing_time" in 5 different places, can we replace it with a constant? 
P.S. it is relevant to outer strings.

Reply:
Its in so many different files

Reply:
which is why in general, we like constants :).  Let's please try and use them in the future.